py/embedded-server/deephaven_server/server.py

```diff
@@ -120,20 +120,38 @@ def __init__(
         port: Optional[int] = None,
         jvm_args: Optional[List[str]] = None,
         extra_classpath: Optional[List[str]] = None,
+        sig_handled_by: Literal["python", "java"] = "python",
     ):
         """
         Creates a Deephaven embedded server. Only one instance can be created at this time.
+
+        Args:
+            host (Optional[str]): The host to bind the server to, defaults to None, meaning local host.
+            port (Optional[int]): The port to bind the server to, defaults to None, meaning 10000.
```

Comment:
This is not necessarily true; it will pick up values from the user's configuration file if present. And that's the primary way that I use it. For reference:

```
$ cat ~/.config/deephaven/deephaven.prop 
includefiles=dh-defaults.prop

http.port=8443
http.http1=false

ssl.identity.type=privatekey
ssl.identity.certChainPath=/home/devin/.config/deephaven/dev-certs/server.chain.crt
ssl.identity.privateKeyPath=/home/devin/.config/deephaven/dev-certs/server.key

outbound.ssl.trust.type=certs
outbound.ssl.trust.path=/home/devin/.config/deephaven/dev-certs/ca.crt

AuthHandlers=io.deephaven.auth.AnonymousAuthenticationHandler
```

Reply:
If the port isn't here, the API feels disjointed.  Half of the connection is a function arg, and the other half is a property specified somewhere else.